ttnn/cpp/ttnn/operations/moreh/moreh_linear/moreh_linear.hpp

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ttnn/cpp/ttnn/operations/core/compute_kernel/compute_kernel_config.hpp"
+#include "ttnn/decorators.hpp"
+
+namespace ttnn::operations::moreh::moreh_linear {
+struct MorehLinear {
+    static Tensor invoke(
+        const Tensor& input,
+        const Tensor& weight,
+        const std::optional<Tensor>& bias,
+        std::optional<Tensor> output,
+        const std::optional<MemoryConfig>& output_mem_config,
```

Comment:
Should be `memory_config`

Reply:
resolved in [3f363ae](https://github.com/tenstorrent/tt-metal/pull/12812/commits/3f363ae0ef41cd058967757ab165f68c13da2d26)